package.json

```diff
@@ -70,6 +70,11 @@
     "ts-node": "^8.10.1",
     "typescript": "^4.3.5"
   },
+  "browserslist": [
+    ">0.2%",
+    "not dead",
+    "not op_mini all"
+  ],
```

Comment:
This fixes the Legacy Javascript lighthouse violation. This project uses CRA which takes advantage of `browserslist` for removing legacy code. By adding a stricter config, only code supporting the defined browsers will be included in your bundle.